docs/docs-content/vm-management/vm-migration-assistant/migrate-vms-vmo-cluster.md

```diff
@@ -0,0 +1,929 @@
+---
+sidebar_label: "Migrate VMs to a VMO cluster"
+title: "Migrate VMs to a VMO cluster"
+description: "Learn how to migrate VMs to Palette VMO using the VM Migration Assistant"
+icon: " "
+hide_table_of_contents: false
+sidebar_position: 20
+tags: ["vmo", "vm migration assistant"]
+---
+
+Follow this guide to create source providers and migrate your VMs to your VMO cluster.
+
+## Limitations
+
+- You can only migrate VMs hosted in VMware vSphere.
+- Only VMs whose operating systems are included under
+  [`virt-v2v` supported guest systems](https://libguestfs.org/virt-v2v-support.1.html) can be migrated.
+
+## Prerequisites
+
+- A healthy VMO cluster. Refer to the [Create a VMO Profile](../create-vmo-profile.md) for further guidance.
+
+  - The VMO cluster must have access to VMware and the VMs you want to migrate.
+
+  :::warning
```

Comment:
This complete warning section already exists in one of the regular VMO page, no? Maybe we could just link to there?